.github/workflows/ubuntu.yml

```diff
@@ -12,16 +12,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [ 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.11.1 ]
+        ruby: [ 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.11.1, truffleruby-head ]
```

Comment:
Should we test with the stable version of truffleruby?

Reply:
The latest release doesn't pass the tests, so it's not an option.

I'll make sure to run RubyGems tests in TruffleRuby's CI too (already the case, but some tests are excluded and no longer need to be).
TruffleRuby's `master` (and therefore `truffleruby-head`) is only updated if the entire TruffleRuby CI passes, so it should not cause failures.

Reply:
I see. We can add the stable version of TruffleRuby after releasing from the current master version.